jekyll/_cci2/ios-tutorial.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: classic-docs
+title: iOS Project Tutorial
+short-title: iOS Project Tutorial
+categories: [migration]
+description: Setting Up iOS applications on CircleCI
+order: 70
+---
+
+This document describes how to automate builds, testing, and deployment of an iOS application project with CircleCI in the following sections:
+
+* TOC
+{:toc}
+
+## Overview
+
+The following sections walk through how to write Jobs and Steps that use `xcodebuild` for this application, how to set up code signing and a provisioning profile in the CircleCI environment, and how to deploy with Fastlane.
+
+## Prerequisites
+
+- Add your project to CircleCI, see [Hello World]( {{ site.baseurl }}/2.0/hello-world/)
+- Enable macOS for your project by going to the Project Settings > Build Environment page of the CircleCI app and enabling the Build OS X project setting. ![Build Environment Settings](  {{ site.baseurl }}/assets/img/docs/ios-getting-started-build-env.png)
+- This tutorial assumes you have an Xcode workspace for your project with at least one shared scheme and that the selected scheme has a test action. If you do not already have a shared scheme, you can add this in Xcode by completing the following steps:
+
+1. Open your Xcode workspace or project. 
+2. Use the scheme selector to open the Manage Schemes dialogue box as shown in the following image.
+![Xcode Scheme Selector](  {{ site.baseurl }}/assets/img/docs/ios-getting-started-scheme-selector.png)
+3. In the manage schemes dialog, select the scheme you wish to build, and ensure that the Shared checkbox is enabled.
+![Manage Schemes Dialogue](  {{ site.baseurl }}/assets/img/docs/ios-getting-started-manage-schemes.png)
+4. Commit and push the schemes.
+
+## Build Phases
+
+In this tutorial, builds are divided into dependencies, test, and deployment. The dependencies phase is for installing any Ruby Gems, CocoaPods, Node Modules or other packages required to build the application. The test phase is for building and testing your project.
+
+For iOS projects, CircleCI generates a command to build and test your project using the `xcodebuild` command line tool. The generated command is similar to:
```

Comment:
We don’t do inference in 2.0. so we should remove this sentence.

The current recommended default is `fastlane scan` instead of `xcodebuild`, we added an example of using that [here](https://circleci.com/docs/2.0/ios-migrating-from-1-2/)